frontend/src/core/hotkeys/hotkeys.ts

```diff
@@ -104,6 +104,11 @@ const DEFAULT_HOT_KEY = {
     group: "Editing",
     key: "Mod-b",
   },
+  "cell.markdownMode": {
```

Comment:
Instead of only switching to "markdown mode", how about using this hotkey to toggle the markdown view on and off? i.e., To switch back and forth between markdown view and Python view?

Reply:
yes! done